.circleci/images/Dockerfile

```diff
@@ -0,0 +1,126 @@
+# Base heroku-16 stack image
```

Comment:
@thermistor this file should be deleted now that we're using other CircleCI image, yeah?